apps/playground/src/app/components/indicators/wait/wait.component.ts

```diff
@@ -1,12 +1,52 @@
-import { Component } from '@angular/core';
+import { ChangeDetectorRef, Component } from '@angular/core';
+import { SkyWaitService } from '@skyux/indicators';
 
 @Component({
   selector: 'app-wait',
   templateUrl: './wait.component.html',
-  styleUrls: ['./wait.component.scss'],
```

Comment:
The `scss` file here was disabling the animations - this was a carry over from it being copied from our old repo where it was disabled for visual tests. Since it was the only css - I just removed the file all together.